Gemfile.lock

```diff
@@ -159,7 +159,7 @@ DEPENDENCIES
   webmock
 
 RUBY VERSION
-   ruby 2.6.3p62
+   ruby 2.7.2p137
 
 BUNDLED WITH
-   1.17.2
+   2.1.4
```

Comment:
Better to just get rid of BUNDLED WITH completely, right?

Reply:
I'm torn; on one hand, we could be liberal about it and say everyone bring your own `bundler`, but that will eventually bite us. I initially removed the line, but Bundler put it back in so I'd suggest we pin the version (i.e keep the line) but make sure it's 2.x.